.github/CONTRIBUTING.md

```diff
@@ -110,6 +122,18 @@ In case you are suggesting a new feature, we will match your idea with our curre
   yarn test
   ```
 
+* Run CLI tests with:
+  
+  ```bash
+  yarn test:cli`
+  ```
+
+* Run package tests with:
```

Comment:
`package`  => `packages`

Reply:
Updated the entire message, looks better now I think